### ISSUE TITLE
Can`t change the senderId

### ISSUE THREAD

**Issue by kimorkim** [MatthieuLemoine/electron-push-receiver#2]
first, i checked the running with the test senderId.
``` javascript
var senderId = '00000001';
ipcRenderer.send(START_NOTIFICATION_SERVICE, senderId);
```
then it was changed to the senderId that was actually used.
``` javascript
var senderId = '00000002';
ipcRenderer.send(START_NOTIFICATION_SERVICE, senderId);
```
but, it did not work.
Because the senderId has not changed. (still old credentials [link](https://github.com/MatthieuLemoine/electron-push-receiver/blob/master/src/index.js#L41))
of course, i can delete the config and solve them.
however, i believe that changing senderId should be easier for various situations.
what do you think about this?

**Comment by MatthieuLemoine** [comment]
Yeah definitely. It should be easier to change the senderId.

**Comment by MatthieuLemoine** [comment]
Released in [v1.2.0](https://github.com/MatthieuLemoine/electron-push-receiver/releases/tag/v1.2.0)
